test/Swashbuckle.AspNetCore.SwaggerGen.Test/Annotations/XmlCommentsMemberNameHelperTests.cs

```diff
@@ -25,7 +26,7 @@ public XmlCommentsMemberNameHelperTests(ITestOutputHelper output)
         [InlineData(typeof(XmlAnnotatedType), "AcceptsArrayOfConstructedGenericType",
             "M:Swashbuckle.AspNetCore.SwaggerGen.Test.XmlAnnotatedType.AcceptsArrayOfConstructedGenericType(System.Nullable{System.Int32}[])")]
         [InlineData(typeof(XmlAnnotatedGenericType<int,string>), "AcceptsTypeParameters",
-            "M:Swashbuckle.AspNetCore.SwaggerGen.Test.XmlAnnotatedGenericType`2.AcceptsTypeParameters(System.Int32,`0,`1)")]
+            "M:Swashbuckle.AspNetCore.SwaggerGen.Test.XmlAnnotatedGenericType`2.AcceptsTypeParameters(System.Int32,System.Int32,System.String)")]
```

Comment:
This is not the desired behavior. You've changed the expected member name to a value that can't possibly be found in an XML comments file. `XmlAnnotatedGenericType<int, string>` is derived from a generic type. It _doesn't_ have it's own class definition and therefore doesn't have it's own XML comments.

If a controller is constructed from a generic type then the desired behavior is to use the XML comments from the generic type because XML comments for the specific type simply won't be available. This feature was implemented to support the common pattern of CRUD actions via a generic base controller:

```csharp
public abstract class GenericResourceController<TResource> where TResource : new()
{
    /// <summary>
    /// creates a resource
    /// </summary>
    /// <param name="resource">The resource representation</param>
    /// <returns></returns>
    [HttpPost]
    public int Create([FromBody, Required]TResource resource)
    {
        return 1;
    }

    ...
}

[Route("orders")]
public class OrdersController : GenericResourceController<Order>
{ }

[Route("products")]
public class ProductsController : GenericResourceController<Product>
{ }
```

In it's current state, this PR breaks this functionality.